package/network/services/hostapd/src/src/ap/ubus.c

```diff
@@ -1053,7 +1065,18 @@ int hostapd_ubus_handle_event(struct hostapd_data *hapd, struct hostapd_ubus_req
 
 	ban = avl_find_element(&hapd->ubus.banned, addr, ban, avl);
 	if (ban)
-		return WLAN_STATUS_AP_UNABLE_TO_HANDLE_NEW_STA;
+	{
+		if(0 < ban->max_ban_attempts && ban->con_tries >= ban->max_ban_attempts)
```

Comment:
missing space after if so it should be `if () {`

Reply:
That logic seems to be weird and quite hairy as well. You should probably make `max_ban_attempts` unsigned int and test here for `if (ban->max_ban_attempts > 0 && ban->con_tries >= ban->max_ban_attempts)` which probably means, that con_tries should be unsigned int as well.